README.md

```diff
@@ -1,5 +1,7 @@
 # Hybrid Custody
 
+![Tests](https://github.com/onflow/hybrid-custody/actions/workflows/integration-tests.yml/badge.svg) [![codecov](https://codecov.io/gh/onflow/hybrid-custody/branch/main/graph/badge.svg?token=5GWD5NHEKF)](https://codecov.io/gh/onflow/hybrid-custody)
```

Comment:
**Note:** The token for the `Codecov` badge has to be generated accordingly from the settings of the repository, in the `Codecov` platform.

Reply:
Relevant documentation: https://docs.codecov.com/docs/status-badges